### DISCUSSION TITLE
Termux packages 90+% up to date!

### DISCUSSION THREAD

**Discussion by finagolfin** [termux/termux-packages#13664] [Announcements]
We are [now back in the top 10 list of most up-to-date package repositories at repology.org](https://repology.org/) (scroll down to the last top 10 list, where Termux is currently fourth). Thanks to @xtkoba for all the work he put in over the last month, and others like @truboxl and so on who contributed. Let's try to keep [these packages up to date](https://repology.org/repository/termux), especially by adding more of them to the auto-updater over time.

**Comment by 2096779623** [comment]
It seems that we should turn on automatic updates for some frequently updated packages, and update outdated packages.
